lib/java_buildpack/container/tomee/tomee_instance.rb

```diff
@@ -34,6 +44,21 @@ def tomcat_7_compatible
         @version < TOMEE_7
       end
 
+      private
+
+      def web_inf_lib
+        if ear?
+          # copy additional libraries to tomee lib folder
+          tomcat_lib
```

Comment:
What will happen if the application supplies it's own version of the driver? Will it use it or it will use the one located in TomEE/lib folder?
I can think of the following scenarios where the application wants to work with a concrete version of the driver:
- Web module packages the driver in WEB-INF/lib and places resources.xml in WEB-INF
- EJB module and Web module use common driver placed in EAR/lib; EJB module places resources.xml in it's META-INF (not the EAR's META-INF) and Web module places resources.xml in WEB-INF
- EJB module uses driver placed in the EAR file and specified in the EJB's META-INF/MANIFEST.MF via Class-Path header; EJB module places resources.xml in it's META-INF (not the EAR's META-INF)